env.example.yml

```diff
@@ -2,4 +2,6 @@ LAMBDA_EXEC_SG: Insert AWS Security Group ID Here (it must be in the same VPC as
 LAMBDA_EXEC_SUBNET: Insert AWS Subnet ID Here (it must be in the same VPC as the security group)
 BUGSNAG_NOTIFIER_KEY: INSERT BUGSNAG NOTIFIER KEY HERE
 GEOCODERS: <Stringified JSON Array of OTP-UI `GeocoderConfig`s>
-BACKUP_GEOCODERS: <Stringified JSON Array of OTP-UI `GeocoderConfig`'s. Same length and order as GEOCODERS> 
+BACKUP_GEOCODERS: <Stringified JSON Array of OTP-UI `GeocoderConfig`'s. Same length and order as GEOCODERS>
+
+LOCATION_COMPARISON_COORDIATE_ACCURACY: defaults to 4 (~10m). What accuracy to use when comparing if two locations are the same
```

Comment:
I think this is actually about the precision of the comparison rather than accuracy, right? It might also be good to mention in this comment what this value is and how it relates to meters. (10^x degrees to meters I guess?)

Reply:
And what do you think? Should this variable be renamed to precision?

Reply:
Yes this should be precision. also note to myself `COORDINATE` is misspelled